### ISSUE TITLE
About the F1_PC_CP_avg

### ISSUE THREAD

**Issue by WxZy530** [why2011btv/JointConstrainedLearning#4]
Sorry to bother,but it confuses me that whether I'm on the right way.I use finetune with the command "python3 main.py gpu_2 batch_16 0.0000001 0920_2.rst epoch_40 HiEve add_loss_1 finetune_1".Below is my result,but get a really low  F1_PC_CP_avg.Is this same to your result?

======== Epoch 35 / 40 ========
Training...
  Batch    40  of    626.    Elapsed: 0:00:16.
  Batch    80  of    626.    Elapsed: 0:00:33.
  Batch   120  of    626.    Elapsed: 0:00:49.
  Batch   160  of    626.    Elapsed: 0:01:06.
  Batch   200  of    626.    Elapsed: 0:01:22.
  Batch   240  of    626.    Elapsed: 0:01:39.
  Batch   280  of    626.    Elapsed: 0:01:55.
  Batch   320  of    626.    Elapsed: 0:02:12.
  Batch   360  of    626.    Elapsed: 0:02:28.
  Batch   400  of    626.    Elapsed: 0:02:45.
  Batch   440  of    626.    Elapsed: 0:03:01.
  Batch   480  of    626.    Elapsed: 0:03:18.
  Batch   520  of    626.    Elapsed: 0:03:34.
  Batch   560  of    626.    Elapsed: 0:03:50.
  Batch   600  of    626.    Elapsed: 0:04:07.

  Total training loss: 2003.88
  Training epoch took: 0:04:18

Running Evaluation on Validation Set...
Eval took: 0:01:58
              precision    recall  f1-score   support

           0       0.08      0.57      0.15       378
           1       0.14      0.42      0.21       465
           2       0.07      0.38      0.11       154
           3       0.00      0.00      0.00      3798

    accuracy                           0.10      4795
   macro avg       0.07      0.34      0.12      4795
weighted avg       0.02      0.10      0.04      4795

  F1_PC_CP_avg: 0.180
